.github/workflows/hygiene.yml

```diff
@@ -0,0 +1,61 @@
+name: Hygiene
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+
+jobs:
+  hygiene:
+    name: Checks
+    runs-on: ubuntu-latest
+    steps:
+#      - name: GitHub Context
+#        run: echo $GITHUB_CONTEXT
+#        env:
+#          GITHUB_CONTEXT: ${{ toJson(github) }}
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+
+      - name: Changes updated
+        run: >-
+          tools/ci/actions/changes.sh
+          'pull_request'
+          '${{ github.event.pull_request.base.ref }}'
+          '${{ github.event.pull_request.base.sha }}'
+          '${{ github.event.pull_request.head.ref }}'
+          '${{ github.event.pull_request.head.sha }}'
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'no-change-entry-needed') && github.event_name == 'pull_request' }}
```

Comment:
could you break a line just before `&&` by using a `>-` multiline?

Reply:
Possibly, although I never confess fighting the YAML parser... I don't think you're allowed extra whitespace, so I personally find it less clear:

```
if: ${{ !contains(github.event.pull_request.labels.*.name, 'no-change-entry-needed')
&& github.event_name == 'pull_request' }}
```

Reply:
Are you sure that indenting the `&&` with the `!` would not work? (I don't see why it would not.)

Reply:
It turns out you can - I did not expect it would work because the `run:` lines above do not work if you indent the parameters (which looks neater). Let's hope that's not down to brittle parsing rules 🙂